### PR TITLE
Content mgmt - Updated article

### PR DESCRIPTION
**Pull request by teizenman**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: content-repo/extra-docs/articles/content-management.md [hunk 1]
```diff
@@ -9,6 +9,58 @@ This process encapsulates what you need in order to control your XSOAR machines
 
 ---
 
+## Building The Repo
+You can visit our [example repo](https://github.com/demisto/content-helloworld-premium) and clone it to use as a base to your repo.
```
Comment (yaakovi) [inline content-repo/extra-docs/articles/content-management.md:13]:
```suggestion
You can visit our [example repository](https://github.com/demisto/content-helloworld-premium) and clone it to use as a baseline.
```

File: content-repo/extra-docs/articles/content-management.md [hunk 2]
```diff
@@ -9,6 +9,58 @@ This process encapsulates what you need in order to control your XSOAR machines
...
+├── README.md
+├── xsoar_config.json
+├── .private-repo-settings
+├── .demisto-sdk-conf
+├── requirements.txt
+├── tox.ini
+├── demistomock.py                    # Can be copied from the Content repo
+├── demistomock.ps1                   # Can be copied from the Content repo
+├── CommonServerPython.py             # Can be copied from the Content repo
+├── CommonServerPowerShell.ps1        # Can be copied from the Content repo
```
Comment (yaakovi) [inline content-repo/extra-docs/articles/content-management.md:56]: are you sure those are needed?

File: content-repo/extra-docs/articles/content-management.md [hunk 3]
```diff
@@ -21,6 +73,59 @@ Consists of the following sections:
 
 ---
 
+## Building the CI/CD process
+
+### Recommneded Steps:
+1. Prepare the enviornment and the virtual enviornment to run demisto-sdk on.
+2. Create an ID set for the private repo, and merge it with Content repo's ID set.
```
Comment (yaakovi) [inline content-repo/extra-docs/articles/content-management.md:80]: how? add the relevant command or a link to the step in the content build (link should use a specific commit)

**Review by yaakovi — APPROVED** [review]

File: content-repo/extra-docs/articles/content-management.md (hunk 2 above)
Comment (teizenman) [reply content-repo/extra-docs/articles/content-management.md:56]: I wasn't able to do that without it.

**Pull request merged**